dev-libs/libite/libite-2.0.0-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="Libite is a collection of useful BSD APIs"
+HOMEPAGE="https://github.com/troglobit/libite"
+SRC_URI="https://github.com/troglobit/libite/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs"
+
+src_configure(){
+	econf --enable-static=$(usex static-libs)
+}
+
+src_prepare(){
+	eapply_user
+	epatch "${FILESDIR}/${P}-fix-which-path.patch"
+	epatch "${FILESDIR}/${P}-fix-path.patch"
```

Comment:
Please use PATCHES variable instead of epatch and then you can remove src_prepare.

Reply:
@Moiman thanks for the comments!